### DISCUSSION TITLE
New Feature Suggestions

### DISCUSSION THREAD

**Discussion by woelrutie** [ScriptedBits/Emulator-Auto-downloads#7] [Ideas]
I have a few suggestions, but I want to first say that your project is already excellent, and these are simply potential improvements:

1. I believe you've previously mentioned this, but I'll echo that it'd be helpful to have an option to automatically extract the downloads and subsequently delete the archives.
2. It'd be helpful if currently installed emulator versions are detected and there's an option to update each one-by-one.
3. I'd like to see the option to select multiple emulators - with version 4.0 of your project, I believe that you can either select one emulator or all, but there is no in-between.
4. The ability to set relative path via the GUI would be nice. As it stands, you're able to select a folder path via the GUI, but for setting a relative path, I had to open the ead.ini file and set the relative path "emupath = ./" to install in the current directory.

**Comment by ScriptedBits** [comment]
Here are my thoughts:
 1. I am looking into this.
 2. When you download an emulator or download all it compares what is already downloaded and only grabs the updates.
 3. Didn't think of this one but I will see if that could be added.  
 4. Not sure on this on because in v4 you can pick what ever directory you like and the option to change it is in the gui.

**Comment by ScriptedBits** [comment]
@woelrutie quick update.
In the v5.0 your suggestion #3 is implemented.  #4 should be working as well in the v5.0 Pre-Release.  Please give it a look and let me know if these looks better to you.  Thanks
